opencti-platform/opencti-graphql/src/schema/stixRefRelationship.ts

```diff
@@ -88,6 +96,7 @@ export const operatingSystems: RelationRefDefinition = {
   stixName: 'operating_system_refs',
   mandatoryType: 'no',
   multiple: true,
+  datable: true,
```

Comment:
Utility of datable if always true?

Reply:
refs that used to be called Meta are not datable